readme.md

```diff
@@ -194,9 +194,9 @@
 - [Backbone](https://github.com/sadcitizen/awesome-backbone#readme) - App framework.
 - [HTML5](https://github.com/diegocard/awesome-html5#readme) - Markup language used for websites & web apps.
 - [SVG](https://github.com/willianjusten/awesome-svg#readme) - XML-based vector image format.
-- [Canvas](https://github.com/raphamorim/awesome-canvas#readme)
-- [KnockoutJS](https://github.com/dnbard/awesome-knockout#readme)
-- [Dojo Toolkit](https://github.com/petk/awesome-dojo#readme)
+- [Canvas](https://github.com/raphamorim/awesome-canvas#readme) - HTML5 and JavaScript charting library.
```

Comment:
Looking at the contents of the list now, this description is not even correct. The list is about the browser Canvas feature.